packages/contracts-bedrock/src/cannon/MIPS.sol

```diff
@@ -44,7 +45,7 @@ contract MIPS is ISemver {
 
     /// @notice The semantic version of the MIPS contract.
     /// @custom:semver 1.0.1
-    string public constant version = "1.0.1";
+    string public constant version = "1.1.0-beta.1";
```

Comment:
Want to double-check this is the right way to update the version?

Reply:
A patch version bump to `1.0.2` will suffice.

Reply:
It's fine to keep bumping the minor version each time we make a change.

Reply:
Was trying to do the versioning based on [this design doc](https://github.com/ethereum-optimism/design-docs/pull/21/files#diff-795d5b7fbc0cd18fb927e94c45479b5559c19433211bca035df451e8bc8d37b5R46), though I guess it isn't officially adopted yet...

Reply:
Your diff is correct with the `beta.1` suffix